Endangered Waves/information/InformationViewController.swift

```diff
@@ -109,7 +112,7 @@ extension InformationViewController {
         case 3:
             switch indexPath.row {
             case 0: // App Survey
-                let url  = URL(string: Constants.appSurveyURL)!
+                let url  = URL(string: "https://www.savethewaves.org/appsurvey/")!
```

Comment:
This URL needs to change in `Constants.swift` as it impacts `AppCordinator.swift` as well.

Reply:
Hello @mmorey,

Thanks for suggesting the survey url fix. I've edited that, and also fixed the oversized icon display.

That was present before my changes, so I assumed it was from an incorrect sized file in firebase. This is for the world surfing reserve in Playa Hermosa. I fixed the icon by resizing it to 48px, but I'm not sure if that's necessary with clean data. That specific icon still displays incorrectly as transparent and without a border. 

![simulator_screenshot_01BF4B8B-7269-476C-941B-C3FF8EDB5BBD](https://user-images.githubusercontent.com/17527761/223177881-52c353bc-dac2-45ba-805c-833499b374e5.png)